src/main/java/com/autotune/common/bulk/BulkServiceValidation.java

```diff
@@ -0,0 +1,95 @@
+package com.autotune.common.bulk;
```

Comment:
@khansaad Thanks for raising the validations PR. Please kindly add the Copyrights header, as it's a new file.